### ISSUE TITLE
Header white Color Text on white color background 

### ISSUE THREAD

**Issue by caTenschert** [owncloud/contacts#357]
### Steps to reproduce
1. open the app
2. selecting a contact
### Expected behaviour

show the contact information First and Name
### Actual behaviour

if the collor of the contact icon is white you cant see the text becoause its white too
most contacts have color white, create bevore oc 7
### Server configuration

debian 8 amd64
apache2                    2.4.10-10+deb8u4   amd64              Apache HTTP Server
mysql-server-5.5           5.5.47-0+deb8u1    amd64
php5                       5.6.19+dfsg-0+deb8 all
ownCloud 9.0.1 (stable))
Contacts  1.1.0.0 ((git pull 16-14-13 12:30)
Update from 8.x
**Signing status (ownCloud 9.0 and above):**

```
No errors have been found.
```

**List of activated apps:**
Enabled:                                                                                                                          
- activity: 2.2.1  
- calendar: 1.1  
- comments: 0.2  
- contacts: 1.1.0.0  
- dav: 0.1.6  
- external: 1.2  
- federatedfilesharing: 0.1.0  
- federation: 0.0.4  
- files: 1.4.4  
- files_external: 0.5.2  
- files_pdfviewer: 0.8  
- files_sharing: 0.9.1  
- files_texteditor: 2.1  
- files_trashbin: 0.8.0  
- files_versions: 1.2.0  
- files_videoplayer: 0.9.8  
- gallery: 14.5.0  
- notifications: 0.2.3  
- ownbackup: 0.3.8  
- provisioning_api: 0.4.1  
- systemtags: 0.2  
- tasks: 0.9.0  
- templateeditor: 0.1  
- updatenotification: 0.1.0 

**The content of config/config.php:**

```
{                                                                                                                                 
    "system": {                                                                                                                   
        "instanceid": "occff9d48b11",                                                                                             
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",                                                                          
        "trusted_domains": [                                                                                                      
            "***REMOVED SENSITIVE VALUE***",                                                                                                  
            "***REMOVED SENSITIVE VALUE***",                                                                                                      
            "***REMOVED SENSITIVE VALUE***",                                                                                                  
            "192.168.178.2",                                                                                                      
            "s0",                                                                                                                 
            "***REMOVED SENSITIVE VALUE***",                                                                                                      
            "192.168.178.202"                                                                                                     
        ],                                                                                                                        
        "datadirectory": "\/var\/oc_data",                                                                                        
        "dbtype": "mysql",                                                                                                        
        "version": "9.0.1.3",                                                                                                     
        "dbname": "owncloud",                                                                                                     
        "dbhost": "localhost",                                                                                                    
        "dbtableprefix": "oc_",                                                                                                   
        "dbuser": "***REMOVED SENSITIVE VALUE***",                                                                                
        "dbpassword": "***REMOVED SENSITIVE VALUE***",                                                                            
        "appstoreenabled": true,                                                                                                  
        "apps_paths": [                                                                                                           
            {                                                                                                                     
                "path": "\/var\/owncloud\/apps",                                                                                  
                "url": "\/apps",                                                                                                  
                "writable": true                                                                                                  
            },                                                                                                                    
            {                                                                                                                     
                "path": "\/var\/owncloud\/apps2",                                                                                 
                "url": "\/apps2",                                                                                                 
                "writable": true                                                                                                  
            }                                                                                                                     
        ],                                                                                                                        
        "installed": true,                                                                                                        
        "theme": "",                                                                                                              
        "maintenance": false,                                                                                                     
        "loglevel": "4",                                                                                                          
        "secret": "***REMOVED SENSITIVE VALUE***",                                                                                
        "forcessl": true,                                                                                                         
        "appstore.experimental.enabled": true,                                                                                    
        "trashbin_retention_obligation": "auto",                                                                                  
        "updater.secret": "***REMOVED SENSITIVE VALUE***"                                                                         
    }                                                                                                                             
}  

or 

Insert your config.php content here
(Without the database password, passwordsalt and secret)
```

too mutch

**Are you using external storage, if yes which one:** local/smb/sftp/...
ftp/smb
**Are you using encryption:** 
no
**Are you using an external user-backend, if yes which one:**
no
### Client configuration

**Browser:**
Chrome 49.0.2623.112 m
**Operating system:**
Windows 7 + 10
**CardDAV-clients:**
CardDAV-Sync 0.4.19 Android from Marten Gajda
#### Web server error log

```
Insert your webserver log here
```
#### ownCloud log (data/owncloud.log)

```
Insert your ownCloud log here
```

{"reqId":"N4dXqHgrkIrZzrhPZYYt","remoteAddr":"x","app":"contacts","message":"Could not find resource file \"\/owncloud
\/apps2\/contacts\/js\/vendor\/angular\/angular.js\"","level":3,"time":"2016-04-13T11:02:39+00:00","method":"GET","url":"\/ownclou
d\/apps\/contacts\/","user":"carl"}  
{"reqId":"N4dXqHgrkIrZzrhPZYYt","remoteAddr":"x","app":"contacts","message":"Could not find resource file \"\/owncloud
\/apps2\/contacts\/js\/vendor\/angular-route\/angular-route.js\"","level":3,"time":"2016-04-13T11:02:39+00:00","method":"GET","url
":"\/owncloud\/apps\/contacts\/","user":"carl"}  
{"reqId":"N4dXqHgrkIrZzrhPZYYt","remoteAddr":"X","app":"contacts","message":"Could not find resource file \"\/owncloud
\/apps2\/contacts\/js\/vendor\/angular-uuid4\/angular-uuid4.js\"","level":3,"time":"2016-04-13T11:02:39+00:00","method":"GET","url
":"\/owncloud\/apps\/contacts\/","user":"carl"}  
{"reqId":"N4dXqHgrkIrZzrhPZYYt","remoteAddr":"X","app":"contacts","message":"Could not find resource file \"\/owncloud
\/apps2\/contacts\/js\/vendor\/angular-cache\/dist\/angular-cache.js\"","level":3,"time":"2016-04-13T11:02:39+00:00","method":"GET
","url":"\/owncloud\/apps\/contacts\/","user":"carl"}  
{"reqId":"N4dXqHgrkIrZzrhPZYYt","remoteAddr":"x","app":"contacts","message":"Could not find resource file \"\/owncloud
\/apps2\/contacts\/js\/vendor\/angular-sanitize\/angular-sanitize.js\"","level":3,"time":"2016-04-13T11:02:39+00:00","method":"GET
","url":"\/owncloud\/apps\/contacts\/","user":"carl"}  
{"reqId":"N4dXqHgrkIrZzrhPZYYt","remoteAddr":"x","app":"contacts","message":"Could not find resource file \"\/owncloud
\/apps2\/contacts\/js\/vendor\/ui-select\/dist\/select.js\"","level":3,"time":"2016-04-13T11:02:39+00:00","method":"GET","url":"\/
owncloud\/apps\/contacts\/","user":"carl"}  
{"reqId":"N4dXqHgrkIrZzrhPZYYt","remoteAddr":"x","app":"contacts","message":"Could not find resource file \"\/owncloud
\/apps2\/contacts\/js\/vendor\/vcard\/src\/vcard.js\"","level":3,"time":"2016-04-13T11:02:39+00:00","method":"GET","url":"\/ownclo
ud\/apps\/contacts\/","user":"carl"}  
{"reqId":"N4dXqHgrkIrZzrhPZYYt","remoteAddr":"x","app":"contacts","message":"Could not find resource file \"\/owncloud
\/apps2\/contacts\/js\/public\/script.js\"","level":3,"time":"2016-04-13T11:02:39+00:00","method":"GET","url":"\/owncloud\/apps\/c
ontacts\/","user":"carl"}  
{"reqId":"N4dXqHgrkIrZzrhPZYYt","remoteAddr":"x","app":"contacts","message":"Could not find resource file \"\/owncloud
\/apps2\/contacts\/js\/vendor\/angular-bootstrap\/ui-bootstrap.min.js\"","level":3,"time":"2016-04-13T11:02:39+00:00","method":"GE
T","url":"\/owncloud\/apps\/contacts\/","user":"carl"}  
{"reqId":"N4dXqHgrkIrZzrhPZYYt","remoteAddr":"x","app":"contacts","message":"Could not find resource file \"\/owncloud
\/apps2\/contacts\/js\/vendor\/angular-bootstrap\/ui-bootstrap-tpls.min.js\"","level":3,"time":"2016-04-13T11:02:39+00:00","method
":"GET","url":"\/owncloud\/apps\/contacts\/","user":"carl"}  
{"reqId":"N4dXqHgrkIrZzrhPZYYt","remoteAddr":"x","app":"contacts","message":"Could not find resource file \"\/owncloud
\/apps2\/contacts\/js\/vendor\/jquery-timepicker\/jquery.ui.timepicker.js\"","level":3,"time":"2016-04-13T11:02:39+00:00","method"
:"GET","url":"\/owncloud\/apps\/contacts\/","user":"carl"}
#### Browser log

```
Insert your browser log here, this could for example include:
Failed to parse SourceMap: https://xxx/owncloud/apps2/contacts/js/vendor/angular-cache/dist/angular-cache.js.map
```

**Comment by DeepDiver1975** [comment]
can we have a screenshot please? THX

**Comment by TecJon** [comment]
![untitled](https://cloud.githubusercontent.com/assets/11375752/14502995/4d0e1fc0-01ae-11e6-86e5-919ea152ed5c.png)

**Comment by TecJon** [comment]
Contact list on the left.
![untitled](https://cloud.githubusercontent.com/assets/11375752/14503049/98228c1c-01ae-11e6-8921-eca2ad3236d3.png)

**Comment by DeepDiver1975** [comment]
What is the uid of the contact? Somthing with an `@` inside?

**Comment by TecJon** [comment]
@DeepDiver1975 correct.
Example: 

```
All%20contacts?uid=20131211T230906.b6438ac334@cloud.example.com
```

This issue exists since contacts 1.1
Side node: the domain in the uid is the OC url of a previous owncloud installation.

**Comment by DeepDiver1975** [comment]
Will be fixed with the next release.

**Comment by gazbarg** [comment]
Same problem here, _except_ the UIDs do not always have an @ inside.

**Comment by fireba11** [comment]
Info: Happens when the contact is created on a client and then synced to ownCloud, does NOT happen when the contact is created in the web gui.

**Comment by DeepDiver1975** [comment]
This is fixed with 1.2

**Comment by fireba11** [comment]
Jup, updated the plguin yesterday and can confirm it's gone.
